spring-xsuaa-it/src/test/java/com/sap/cloud/security/xsuaa/extractor/BasicAuthenticationValidationTest.java

```diff
@@ -1,130 +0,0 @@
-/**
- * SPDX-FileCopyrightText: 2018-2022 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-package com.sap.cloud.security.xsuaa.extractor;
-
-import static org.hamcrest.Matchers.containsString;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
-
-import java.util.Base64;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.mock.web.MockHttpServletRequest;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.test.web.servlet.MockMvc;
-import org.springframework.test.web.servlet.request.RequestPostProcessor;
-
-import com.sap.cloud.security.xsuaa.mock.JWTUtil;
-
-import testservice.api.XsuaaITApplication;
-import testservice.api.basic.SecurityConfiguration;
-import testservice.api.basic.TestController;
-
-@RunWith(SpringRunner.class)
-@SpringBootTest(properties = {
-		"xsuaa.xsappname=java-hello-world",
-		"xsuaa.clientid=sb-java-hello-world" }, classes = { XsuaaITApplication.class, SecurityConfiguration.class,
-				TestController.class })
-@AutoConfigureMockMvc
-@ActiveProfiles("test.api.basic")
-public class BasicAuthenticationValidationTest {
```

Comment:
where do you test Basic auth flow?

Reply:
Integration via Basic Auth is not tested anymore because TokenBrokerResolver will be moved out into sample code only.